sound/soc/bcm/rpi-wm8804-soundcard.c

```diff
@@ -372,6 +474,20 @@ static int snd_rpi_wm8804_probe(struct platform_device *pdev)
 		snd_clk48gpio =
 			devm_gpiod_get(&pdev->dev, "clock48", GPIOD_OUT_LOW);
 
+		led_gpio_1 = devm_gpiod_get(&pdev->dev, "led1", GPIOD_OUT_LOW);
```

Comment:
Board-specific code like this needs to go into the board-specific probe function, in this case snd_interlude_audio_digital_probe.